### DISCUSSION TITLE
SASA

### DISCUSSION THREAD

**Discussion by SpaceTowel** [ISS-Mimic/Mimic#304] [General]
Things we need:
1)    Print parts
2)    (2) 25kg servos (Bryan will bring from home)
3)    Arduino of some kind
4)    Servo shield  [(1](https://www.amazon.com/KEYESTUDIO-16-Channel-12-bit-Shield-Arduino/dp/B0797JK4RW/));  
5)    Power supply
6)    Servo extension
7)    LED,  [small ring](https://www.amazon.com/Adafruit-NeoPixel-Ring-Integrated-Drivers/dp/B00KAE3R1U/ref=sr_1_5?crid=1RUSLH5Q78RUJ&keywords=neopixel+ring&qid=1644445577&sprefix=neopixel+ring%2Caps%2C71&sr=8-5) @streadgold  .  

8) LED 2[nd one](https://www.amazon.com/Adafruit-Flora-Smart-NeoPixel-Version/dp/B00KBXTJRQ/)
9) misc wire

What else can it do:
1) Point at ISS
2) Disco mode
3) SASA mode/screen on PI
4) Direction (true North) on base.  3D print or Vinyl?

**Comment by juggler2000** [comment]
I brought in the 25 kg-cm motors and some SASA parts I printed a while ago. Also brought an extra RasPi with touchscreen.
